net/grpc/gateway/docker/prereqs/Dockerfile

```diff
@@ -61,12 +64,10 @@ COPY --from=prepare /usr/local/bin/protoc /usr/local/bin/
 COPY --from=prepare /usr/local/bin/buildifier /usr/local/bin/
 COPY --from=prepare /github/grpc-web/third_party /github/grpc-web/third_party
 
-RUN wget -nv -O bazel-installer.sh \
-  https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/\
-bazel-$BAZEL_VERSION-installer-linux-x86_64.sh && \
-  chmod +x ./bazel-installer.sh && \
-  ./bazel-installer.sh && \
-  rm ./bazel-installer.sh
+RUN wget -nv -O /usr/local/bin/bazelisk \
+  https://github.com/bazelbuild/bazelisk/releases/download/v$BAZELISK_VERSION/bazelisk-linux-$TARGETARCH && \
```

Comment:
Use of bazelisk is required, since `bazel-$BAZEL_VERSION-installer-linux-$arch` does not exist for arm64

Reply:
Thanks for the explanation!

May i ask you a favor to list out *all* the dependency updates you've made in this PR in a bullet-point summary? (Feel free to mention all other notable changes you made as well!)